types/gravatar/index.d.ts

```diff
@@ -1,23 +1,35 @@
-// Type definitions for gravatar v1.8.0
+// Type definitions for gravatar 1.8
 // Project: https://github.com/emerleite/node-gravatar
 // Definitions by: Denis Sokolov <https://github.com/denis-sokolov>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace GravatarModule {
-  type Options = {
-    d?: string
-    default?: string
-    f?: string
-    forcedefault?: string
-    format?: string
-    protocol?: string
-    r?: string
-    rating?: string
-    s?: string
-    size?: string
-  }
-
-  function url(email: string, options?: Options, forceProtocol?: boolean): string;
+export interface Options {
+    cdn?: string;
+    d?: string;
+    default?: string;
+    f?: string;
+    forcedefault?: string;
+    format?: string;
+    protocol?: string;
+    r?: string;
+    rating?: string;
+    s?: string;
+    size?: string;
 }
-
-export = GravatarModule;
```

Comment:
The previous export defined a CommonJS export, while the new ones pretend to be ES2015 modules. The library itself uses a CommonJS export. I’m not sure, but maybe this might break the typechecking for users who have `esModuleInterop` set to false.

Reply:
I understand your concerns, let me revisit the definition (though I thinks it's ok).

Reply:
While I generally prefer to match the shape of the underlying library, I don't believe it will make a difference in this case because the underlying export isn't invokable.

Reply:
@amcasey thx!
@denis-sokolov let's know your opinion when you have a second!

Reply:
I brought this to your attention to make sure you are aware of it, but I don’t have a concern. `esModuleInterop` is strongly recommended anyway, so it will likely not impact most users.